### DISCUSSION TITLE
Llist of all the Software that can be used with SEB in Mac vs Windows devices

### DISCUSSION THREAD

**Discussion by petya-inspera** [SafeExamBrowser/seb-win-refactoring#1068] [Q&A]
Hello. 

We have received a request from one of our customers asking for a list of all the software that can be used with SEB in MAC and Windows. Can you provide a list?

The same customer reported that Sogou Pinyin software works with SEB on windows but not on MAC. Is this expected? They are also curious to know if the Audacity software is compatible with SEB

**Comment by danschlet** [comment]
There is no list of compatible software and we cannot provide such a list ourselves (we don't have the capacity to test millions of applications with SEB).

But we can create a thread in here in the repositories Discussions section where we and other SEB users can list apps they successfully tested/are using with SEB.

Regarding the software you mentioned, we don't know it. Support for additional third-party apps in the macOS version of SEB is new and we need to update our documentation to explain how to use it. It will also be improved in next releases.

I will post a link to the additional software thread as soon as we create it.
